cmd/operator-verify/main.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import (
+	"github.com/dweepgogia/new-manifest-verification/cmd"
+)
+
+func main() {
+	//Implementing CLI tool using Cobra
```

Comment:
The comment is wrong. I suggest `// Launch CLI tool`

Reply:
Done.